sklearn/inspection/_partial_dependence.py

```diff
@@ -177,9 +186,98 @@ def _partial_dependence_brute(est, grid, features, X, response_method):
     return averaged_predictions
 
 
+def _validate_parameters(estimator, X, features, response_method, method):
+    if not (is_classifier(estimator) or is_regressor(estimator)):
+        raise ValueError(
+            "'estimator' must be a fitted regressor or classifier."
+        )
+
+    if isinstance(estimator, Pipeline):
+        # TODO: to be removed if/when pipeline get a `steps_` attributes
+        # assuming Pipeline is the only estimator that does not store a new
+        # attribute
+        for est in estimator:
+            # FIXME: remove the None option when it will be deprecated
```

Comment:
Added in https://github.com/scikit-learn/scikit-learn/pull/14028/files#diff-0c8623ee957e0256d29ac53830094281R315.

The deprecation seems to be in `FeatureUnion` rather than `Pipeline`. See https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/pipeline.py#L773-L774